helmfile/values/dex.yaml.gotmpl

```diff
@@ -1,4 +1,4 @@
-imageTag: "v2.26.0"
+imageTag: "v2.28.1"
```

Comment:
Did you try this out?

I'm a bit surprised that we need to add something in the `config.storage` key, like [here](https://github.com/dexidp/helm-charts/tree/master/charts/dex#minimal-configuration).